include/gdf/cffi/types.h

```diff
@@ -27,6 +32,7 @@ typedef struct gdf_column_{
     gdf_valid_type *valid;
     gdf_size_type size;
     gdf_dtype dtype;
+    gdf_size_type null_count = 0;
```

Comment:
`cffi` can't handle the zero initializer.

Reply:
ok we can drop that